adhocracy-plus/assets/js/init-picker.js

```diff
@@ -1,15 +1,77 @@
-const $ = require('jquery')
-const datePickerController = require('datepicker')
-const django = require('django')
+import django from 'django'
+import flatpickr from 'flatpickr'
 
-$(function () {
-  const $inputs = $('.datepicker')
+function link (f0, f1) {
+  const date = f0.selectedDates[0]
+  if (date) {
+    // if p0 has a date set that date as minDate for p1
+    f1.set('minDate', f0.formatDate(date, f0.config.dateFormat))
+  }
+  f0.config.onChange.push((selectedDates, dateStr) => {
+    // if date of p0 is changed adapt minDate for p1
+    f1.set('minDate', dateStr)
+  })
+}
+
+function linkDatePickers (flatpickrs) {
+  // normal flatpickers
+  const idStart = 'id_start_date_date'
+  const idEnd = 'id_end_date_date'
+  // (multi-)phase flatpickrs
+  const phaseIds = [
+    'id_phase_set-0-start_date_date',
+    'id_phase_set-0-end_date_date',
+    'id_phase_set-1-start_date_date',
+    'id_phase_set-1-end_date_date',
+    'id_phase_set-2-start_date_date',
+    'id_phase_set-2-end_date_date'
+  ]
+
+  // link non-phase datepickers if exist
+  const fStart = flatpickrs.get(idStart)
+  const fEnd = flatpickrs.get(idEnd)
+  if (fStart && fEnd) {
+    link(fStart, fEnd)
+  }
+
+  // link phase datepickers if exist
+  for (let i = 0; i < phaseIds.length - 1; i++) {
+    if (flatpickrs.length <= i) {
+      return
+    }
+    const p0 = flatpickrs.get(phaseIds[i])
+    const p1 = flatpickrs.get(phaseIds[i + 1])
+    if (p0 && p1) {
+      link(p0, p1)
+    }
+  }
+}
 
-  $inputs.addClass('form-control')
+function initDatePicker () {
```

Comment:
not sure if that's something @rjlanari wants to take over as the focus was more on django, what do yo u think @rjlanari ? Otherwise maybe you can work on that @hom3mad3 if you have an idea in mind? If we change it we should port it over to mb (or if both files are identical we could even consider moving it into a4)